CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 56 Release Notes
 
+### Retry with More Memory as workflow option
+
+Retry with more memory is now customizable via a global config. The multiply factor is passed in through the workflow 
```

Comment:
We should probably note that this is a breaking change and people will now need to:
* Move their `error-keys` to global configuration
* Remember to specify a multiplier in their workflow options

Reply:
( my suggestion should address this )